TODO.md

```
@@ -23,10 +24,36 @@ feat:
   - [ ] console must ask which environments to run
   - [ ] console must ask path
 
-  when remote pasta added must update remote pastas list
+  pastas:
+    - [ ] delete mode: on pasta click make it red-ish and on accept delete red-ish pastas
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)